tests/test_default.py

```diff
@@ -0,0 +1,12 @@
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    '.molecule/ansible_inventory').get_hosts('all')
+
+
+def test_hosts_file(File):
```

Comment:
This is the default example molecule test. Does molecule still run successfully if there aren't any test files?